client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -60,6 +60,70 @@ function PlaceholderPostEdit( props ) {
 	);
 }
 
+function RelatedPostsEditItem( props ) {
+	const previewClassName = 'related-posts__preview';
+
+	return (
+		<div className={ `${ previewClassName }-post` }>
+			{ props.displayThumbnails && props.post.img && props.post.img.src && (
+				<Button className={ `${ previewClassName }-post-link` } isLink>
+					<img src={ props.post.img.src } alt={ props.post.title } />
+				</Button>
+			) }
+			<h4>
```

Comment:
Should we do a check that sees if { props.post.title} even exits?

Reply:
The main issue I dealt with yesterday was with `props.post` becoming `undefined` a couple of times. The `title` should always be defined, but let me double-check that today.